lib/client-assets.php

```diff
@@ -127,7 +127,9 @@ function gutenberg_override_translation_file( $file, $handle ) {
 	}
 
 	// Ignore scripts that are not found in the expected `build/` location.
-	$script_path = gutenberg_dir_path() . 'build/' . substr( $handle, 3 ) . '/index.js';
+	$suffix = ( defined( 'SCRIPT_DEBUG') && SCRIPT_DEBUG ) ? '.js' : '.min.js';
```

Comment:
I'm still confused here and I'm not really sure if we need this ternary logic for the suffix here or if we should hardcode this to the unminized `.js` file, since this function seems related to translations. Any hints?

Reply:
Thanks for explaining, that makes sense 👍🏻

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/31732/commits/8e689a2edf90dc5595a6bfa49c25b72a9af40f2d.